src/authority.rs

```diff
@@ -19,10 +19,12 @@ use xor_name::XorName;
 use sodiumoxide::crypto::{hash, sign};
 use std::fmt::{Debug, Formatter};
 
-/// Persona types recognised by network.
+/// An entity that can act as a source or destination of a message.
+///
+/// An `Authority` can be an individual client or node, or a group of nodes.
```

Comment:
This maybe should be more explicit. So node, NodeManager, NaeManager, ClientManager